src/sardana/tango/macroserver/test/base.py

```diff
@@ -44,7 +44,7 @@ class BaseMacroServerTestCase(object):
     door_name = getattr(sardanacustomsettings, 'UNITTEST_DOOR_NAME',
                         "door/demo1/1")
 
-    def setUp(self, pool_name):
+    def setUp(self, pool_name=None, properties=None):
```

Comment:
Is it difficult to merget both:` pool_name` and `properties` into one properties only? If yes, I think it is worth to do it. Also, all the tests already using `pool_name` shoudl be adapted.